app/uk/gov/hmrc/alcoholdutyreturns/controllers/CacheController.scala

```diff
@@ -62,11 +68,37 @@ class CacheController @Inject() (
           .createUserAnswers(userAnswers)
           .foldF(
             err => Future.successful(error(err)),
```

Comment:
Agreed for the first interation, add should be called rather than any separate endpoint for auditing.
add is the wrong call here there should be a StartReturn controller since the frontend shouldn't be dictating the add, and the audit doesn't really belong in this controller - this can be fixed in a later refactor under this or another ticket.

Reply:
Audit call belongs here but agreed that the method should have been named differently and belongs to a different controller which is out of scope for this ticket. Happy with this approach